app/views/user/update_profile.html.slim

```diff
@@ -29,5 +29,28 @@
       tr
         th =f.label :location
         td =f.text_field :location, :placeholder => 'Where are you from?'
+
+    p.email_notifications Select which of the following notifications you want to receive:
+    table.form.email_notifications
+      tr
+        td =f.check_box("notifications[add_as_collaborator]", {checked: @user.notification.add_as_collaborator})
+        th =f.label :add_as_collaborator, "Added to private work as a collaborator"
+      tr
+        td =f.check_box("notifications[add_as_owner]", {checked: @user.notification.add_as_owner})
+        th =f.label :add_as_owner, "Added to private work as an owner"
+      tr
+        td =f.check_box("notifications[note_added]", {checked: @user.notification.note_added})
+        th =f.label :page_edited, "Note added in response to a comment you made"
+      tr
+        td =f.check_box("notifications[work_added]", {checked: @user.notification.work_added})
+        th =f.label :work_added, "Work added to collection you've worked on"
+      tr
+        td =f.check_box("notifications[page_edited]", {checked: @user.notification.page_edited})
```

Comment:
This one needs to be unchecked by default.

Reply:
Which one needs to be unchecked by default?  There are a couple of different lines highlighted here.

Reply:
page edits.  I don't think you want all page edits emailed to you right away by default.

Reply:
Well, they aren't being immediately emailed under any circumstances - they're in the overnight emails.  Though you may still not want them sent to you by default; what do you think?